scripts/github/deploy.sh

```diff
@@ -67,6 +67,11 @@ echo "Updating version from ${PREVIOUS_VERSION} to ${NEW_VERSION}"
 # Update package files versions, project inter-dependencies and lerna.json with new version number
 lerna version "${NEW_VERSION}" --yes --no-push --force-publish --exact
 
+echo "Updating package-lock.json from ${PREVIOUS_VERSION} to ${NEW_VERSION}"
+# Update package-lock.json files to reflect new versions
+lerna exec -- npm install --package-lock-only
```

Comment:
How's this different to what you did before (not being snarky, genuine question)?

Reply:
change in order and updated echo so same lets know version updated with. Now once package.json updated it will update package-lock straight after with same version then deploy